src/marqo/README.md

```diff
@@ -35,14 +35,56 @@ docker run --detach --name vespa --hostname vespa-tutorial \
 ```bash
 (cd scripts/vespa_local && zip -r - * | curl --header "Content-Type:application/zip" --data-binary @- http://localhost:19071/application/v2/tenant/default/prepareandactivate)
 ```
-
 You can verify that Vespa has been set up correctly by visiting `http://localhost:8080` in your browser.
 
+5. We've written custom logic to handle to reranking of all the results that Vespa outputs. We call this Hybrid search. Hybrid Search combines both lexical and tensor search to provide the best relevance possible. Hybrid search provides flexibility to get the best results of both types of search and fine tune which type to lean into. It's a great place to start when you want the precision of lexical search but the semantic understanding of tensor search.
+The logic is written in Java so you need to create a jar file 
+
+5a. Install Java Development Kit: 
+You can install the Java Development Kit (JDK) by following the instructions [here](https://openjdk.org/install/). 
+You will need to set Java to Path, you can do this by 
+```zsh
+vi ~/.zshrc
+```
+or 
+```zsh
+vi ~/.zprofile
+```
+In the .zshrc/.zprofile file, paste the following contents: 
+```
+JAVA_HOME="/Library/Java/JavaVirtualMachines/jdk-22.0.2.jdk/Contents/Home"
+PATH="${JAVA_HOME}/bin:${PATH}"
+```
+Remember to replace the "/Library/Java/JavaVirtualMachines/jdk-22.0.2.jdk/Contents/Home" with the path to your JDK.
```

Comment:
You could provide links but no need to cover how to install java etc in detail here

Reply:
added links to install java/maven & removed details on installing